src/Makefile

```diff
@@ -266,7 +268,11 @@ ifneq (,$(shell which i686-pc-mingw32-gcc))
 	MINGW_HOST="i686-pc-mingw32"
 else
 ifneq (,$(shell which i686-w64-mingw32-gcc))
-	MING_BASE:=i686-w64-mingw32-
+	ifeq (${CC}, gcc)
+		MING_BASE:=i686-w64-mingw32-
+	else
+		MING_BASE:=
+	endif
 	MINGW_HOST="i686-w64-mingw32"
```

Comment:
Is this solution compatible with all other supported versions of MinGW? You have only changed the case of `i686-w64-mingw32-gcc`.

Reply:
You are right, I did not change the other versions. I am going to change those, too.